kura/examples/org.eclipse.kura.example.ble.tisensortag.dbus/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 1.0.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Service-Component: OSGI-INF/*.xml
 Import-Package: org.eclipse.kura;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[2.0,3.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
```

Comment:
Is it ok to have [1.0,2.0)? I mean, the examples use the apis that were added for example with revision 1.3 or not?